### DISCUSSION TITLE
Questions related to Weaviate on flexibility and offline support?

### DISCUSSION THREAD

**Discussion by Pratyush2758** [Sam-Pewton/weaviate-community.rust#57] [Q&A]
Hi,
I wanted to ask few things related to Weaviate Vector Database.

As an open source vector database, can Weaviate be self hosted ?
Also if needed in the future, will Weaviate be able to provide the Offline support or not? If yes, the supportive links of docs would be really helpful.

**Comment by Sam-Pewton** [comment]
Hi @Pratyush2758 

Apologies for the slow response, I did not see your questions.

I think your questions would be better suited on the Weaviate forum where one of Weaviate's tech team would be able to answer more accurately.

You can find the forum here:
https://forum.weaviate.io/

Alternatively you can also join the Weaviate Slack to ask your questions there. I believe this is the link:
https://weaviate.io/slack
